src/ProgressOnderwijsUtils/ProgressOnderwijsUtils.csproj

```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup Label="Configuration">
-    <Version>5.1.3</Version>
+    <Version>5.1.4</Version>
```

Comment:
misschien wat nitpicky, maar ik zou dit backwards incompatible noemen en bumpen naar 6. Maakt niet heel veel uit aangezien verder niemand t nog gebruikt volgens mij.

Reply:
Ik ben het niet met je oneens, maar volgens mij hebben we nog nooit een backwards-compatibele wijziging gedaan, dus eigenlijk zouden we al op v53 moeten zitten.

Reply:
Er borrelt een retorische vraag in mij

Reply:
Of misschien heb je gewoon te weinig gegeten?

Reply:
We heben zeker weten wel backwards compatible changes gemaakt volgens een redelijke definitie van compatibiliteit.

Als je reflectie mee gaat rekenen kun je semver sowieso vergeten.

Als clients na een update met grote waarschijnlijkheid nog steeds compileren en werken zonder aanpassingen, is dat niet een breaking change (als je iig die term ueberhaupt wilt kunnen hanteren).